cypress/e2e/api/recuperaCliente.cy.js

```diff
@@ -0,0 +1,12 @@
+/// <reference types="cypress" />
+
+describe('Teste de API', () => {
+  it('GET - Recupera clientes com sucesso', () => {
+    cy.request('http://localhost:3001/customers')
+    .should((response)=>{
```

Comment:
Veja outra alternativa para o mesmo teste, só pra você expandir seu conhecimento:

```js
cy.request('GET', CUSTOMERS_API_URL).as('getCustomers')

cy.get('@getCustomers')
  .its('status')
  .should('eq', 200)
```

Onde a `CUSTOMERS_API_URL` estaria definida como uma constante (`const`) no início do arquivo, da seguinte forma: ``const CUSTOMERS_API_URL = `${Cypress.env('API_URL')}/customers``

E o arquivo `cypress.config.js` teria uma estrutura como abaixo:

```js
const { defineConfig } = require('cypress')

module.exports = defineConfig({
  env: {
    API_URL: 'http://localhost:3001',
  },
  e2e: {
    baseUrl: 'http://localhost:3000',
    // qualquer outra configuração para testes e2e aqui.
  },
})
```

Reply:
Outra alternativa, seria desestruturar da `response` só o `status`.

Assim:

```js
.should(({ status })=>{
  expect(status).to.equal(200)
})
```

Para aprender mais, recomendo assistir este vídeo: [JavaScript para QAs #8 - Atribuição via Desestruturação](https://youtube.com/live/yW1JQoz6PXU?feature=share)

Reply:
obrigada pela dica